feat(curriculum): add music player workshop to FSD cert (#57439)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-music-player/67517ef2fae05b65108212db.md

---
id: 67517ef2fae05b65108212db
title: Step 35
challengeType: 0
dashedName: step-35
---

# --description--

You need to get the `id` of the currently playing song. For this, you can use `userData.currentSong?.id`.

> Use `getElementById()` to get the `id` of the currently playing song, then use template literals to prefix it with `song-`. Assign it to a variable named `songToHighlight`.

Reviewer: ```suggestion
Use `getElementById()` to select the currently playing song: pass a template literal to prefix the `id` with `song-`. Assign it to a variable named `songToHighlight`.
```

something like this
